src/ui/settings/ViewAppPermissions.qml

```diff
@@ -36,9 +34,10 @@ Item {
         Component.onCompleted: {
             console.log("Component ready");
             VPNAppPermissions.requestApplist();
-            Sample.appPermissionsViewOpened.record();
+            VPN.recordGleanEvent("appPermissionsViewOpened");
+
             if (!vpnIsOff) {
-                Sample.appPermissionsViewWarning.record();
```

Comment:
Can we enforce this? so that we do not use `Sample` by mistake?

Reply:
Not right now. I've filed https://mozilla-hub.atlassian.net/browse/VPN-2390 - To refactor how we load glean.js. If we were to use it similar as lottie.js, we could remove the global qml sample.